panther_docking/launch/docking.launch.py

```diff
@@ -119,6 +128,20 @@ def generate_launch_description():
         condition=IfCondition(use_docking),
     )
 
+    apriltag_node = Node(
+        package="apriltag_ros",
+        executable="apriltag_node",
+        parameters=[{"use_sim_time": True}, apriltag_config_path],
```

Comment:
Redundancy: In panther_gazebo there already exist `apriltag.launch.py`

Reply:
removed from panther_gazebo.
It has to be a mistake during merge :C